### DISCUSSION TITLE
Different avatar showing in Rainbowkit

### DISCUSSION THREAD

**Discussion by tom-kell** [rainbow-me/rainbowkit#1218] [Feedback]
Hi there, I have a custom avatar showing when I connect rainbowkit and zora which I don't see in my wallet or when connecting to other web3 platforms. I think I might have set this avatar via mirror, but have since changed it. Is there a way to flush out this cache, if that's what it is?

I assume I'd have the option to set an avatar via registering an ens with an avatar??

Thanks

**Comment by DanielSinclair** [comment]
We rely on Wagmi's `useEnsAvatar()` hook for this behavior, which resolves the `avatar` in your ENS Profile. I think you can edit this in the ENS dApp, or in Rainbow mobile.
